packages/server/lambdas/alfresco/searchQueryBuilder/types.ts

```diff
@@ -32,6 +33,10 @@ export type Paging = {
   skipCount: number;
 };
 
+export enum Include {
```

Comment:
"Include" name is a bit confusing. At least I don't understand what its purpose based on the name?

Reply:
That's what the field is called in the API, but it definitely is not descriptive. I'll think of something better. Thanks!

Reply:
Updated to AdditionalFields.